google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -42,6 +42,13 @@ class InstanceAdminEmulator final
     google::protobuf::TextFormat::PrintToString(*request, &request_text);
     std::cout << __func__ << "() request=" << request_text << "\n";
 
+    auto constexpr kMaxInstanceIdLength = 33;
+    auto constexpr kMinInstanceIdLength = 6;
+    if (((request->instance_id()).size() > kMaxInstanceIdLength) ||
```

Comment:
Nit: I don't like all the redundant parentheses, but your choice.

Reply:
Fixed.